common/variable_ticker.go

```diff
@@ -5,19 +5,25 @@ import (
 	"fmt"
 	"math"
 	"time"
+
+	"github.com/Layr-Labs/eigensdk-go/logging"
 )
 
 // Any frequency at or below this value will be interpreted as a frequency of 0 Hz. Needed to avoid overflow.
 // The factor of 2 is to take care of floating point precision issues.
-const MinimumFrequency = float64(time.Nanosecond/math.MaxInt64) * 2.0
+const MinimumFrequency = float64(time.Second/math.MaxInt64) * 2.0
 
 // Any frequency above this value will be interpreted as a frequency of MaximumFrequency Hz. Needed to avoid overflow.
-const MaximumFrequency = float64(time.Nanosecond)
+const MaximumFrequency = float64(time.Second)
```

Comment:
This is kind of a confusing way to define the maximum frequency, any reason not to just do 1e9 directly? I suppose you're tying the max frequency to the time system's precision, but it would be useful to state what this is doing: at first non-critically-thinking glance, this looks like "max frequency is 1 Hz"